content/en/docs/Features/health.md

```diff
@@ -1,18 +1,29 @@
 ---
 title: "Health"
-date: 2018-06-20T19:04:38+02:00
-draft: false
 weight: 2
 ---
 
-## Health
+Kiali help users know whether their service mesh is healthy. This includes the health of the mesh infrastructure itself, and the deployed application services.
 
-Colors in the graph represent the health of your service mesh. A node colored red or orange might need attention. The color of an edge between components represents the health of the requests between those components. The node shape indicates the type of component such as services, workloads, or apps.
+## Service Mesh Infrastructure Health
 
-The health of nodes and edges is refreshed automatically based on the user's preference. The graph can also be paused to examine a particular state, or replayed to re-examine a particular time period.
+Users can quickly confirm the health of their infrastructure by looking at the Kiali Masthead. If Kiali detects any health issues with the infstructure of the mesh it will show an indication in the masthead, severity will be reflected via color, and hovring will show the detail:
```

Comment:
misspell:

```suggestion
Users can quickly confirm the health of their infrastructure by looking at the Kiali Masthead. If Kiali detects any health issues with the infrastructure of the mesh it will show an indication in the masthead, severity will be reflected via color, and hovering will show the detail:
```